app-android/src/main/res/layout/item_clazz_log_attendance_list.xml

```diff
@@ -67,7 +67,7 @@
 
         <!-- Line 1-->
         <TextView
-            android:id="@+id/line1_text"
+            android:id="@+id/report_title"
```

Comment:
Is there a reason that this file (from attendance view) was modified? Seems like just the rename of a textview id.